dhis-2/build-dev.sh

```diff
@@ -29,33 +29,12 @@ print() {
     echo -e "\033[1m$1\033[0m" 1>&2
 }
 
-#
-## The Business
-#
-
-# Requires maven to be on the classpath
-# Skips clean and test phases
-# Also skips copying test resources and compiling tests
-
-print "Building dhis2-core..."
+print "Building dhis2-core and Docker image..."
 
 export MAVEN_OPTS="-Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.test.skip=true"
-mvn clean install --batch-mode --no-transfer-progress --threads 1C -Pdev -f "$DIR"/pom.xml -pl -dhis-web-embedded-jetty
-mvn clean install --batch-mode --no-transfer-progress --threads 1C -Pdev -f "$DIR"/dhis-web/pom.xml
-
-rm -rf "$ARTIFACTS/*"
-mkdir -p "$ARTIFACTS"
-cp -f "$DIR/dhis-web/dhis-web-portal/target/dhis.war" "$ARTIFACTS/dhis.war"
-
-print "Build succeeded, creating Docker image $IMAGE:$TAG..."
-
-cd $ARTIFACTS
-sha256sum ./dhis.war >./sha256sum.txt
-md5sum ./dhis.war >./md5sum.txt
-
-ONLY_DEFAULT=1 $ROOT/docker/build-containers.sh $IMAGE:$TAG $TAG
-
-print "Successfully created Docker image $IMAGE:$TAG"
+mvn clean install --threads 2C -DskipTests -Dmaven.test.skip=true -f dhis-2/pom.xml -pl -dhis-web-embedded-jetty,-dhis-test-integration,-dhis-test-coverage
+mvn clean install --threads 2C -DskipTests -Dmaven.test.skip=true -f dhis-2/dhis-web/pom.xml
+mvn -DskipTests -Dmaven.test.skip=true -f dhis-2/dhis-web/dhis-web-portal/pom.xml jib:dockerBuild
```

Comment:
Should this skip tests?

Reply:
Tests were already skipped before. This is just a helper script to build DHIS2. Its not even referenced in the code.